src/libraries/System.Formats.Tar/src/System.Formats.Tar.csproj

```diff
@@ -0,0 +1,80 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <Nullable>enable</Nullable>
+    <PackageDescription>
+      Provides classes that can read and write the Tar archive format.
+
+      Commonly Used Types:
+      System.Formats.Tar.TarFile
+      System.Formats.Tar.TarEntry
+      System.Formats.Tar.TarReader
+      System.Formats.Tar.TarWriter
+    </PackageDescription>
+  </PropertyGroup>
+  <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
+  <PropertyGroup>
+    <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' == ''">SR.PlatformNotSupported_SystemFormatsTar</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
This can never be true right? We only have 2 TFMs - windows and Unix. There is no empty TargetPlatformIdentifier.

Reply:
I removed the `<GeneratePlatformNotSupportedAssemblyMessage>` line. I did not remove the previous line `<TargetPlatformIdentifier>` because it's being used below, although maybe the value pre-exists, so I don't need to collect the property here? Do I remove it too?

Reply:
My comment was only regarding line 19:

```xml
<GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' == ''">SR.PlatformNotSupported_SystemFormatsTar</GeneratePlatformNotSupportedAssemblyMessage>
```

Line 18 above appears necessary since you use it later.

Reply:
OK, looking deeper, I believe you can remove line 18 as well now. See items 3 & 4 in https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/project-guidelines.md#targetframework-conditions.

You only need to set `<TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>` yourself if you need to condition some properties based on TargetPlatformIdentifier. Since you no longer need to condition the `GeneratePlatformNotSupportedAssemblyMessage` property, and only need to use TargetPlatformIdentifier in an ItemGroup condition, you can just remove it.

Reply:
Thanks! Removing it.